docs/source/notebooks/Extract_geographical_watershed_properties.ipynb

```diff
@@ -324,13 +322,14 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Downloading to /tmp/tmpq5dx3ekj/subset_1.tiff.\n"
+      "Metalink content-type detected.\n",
```

Comment:
Are you on the latest birdy?  Funny that the output has the additional "Metalink content-type detected"?

Reply:
I setup a fresh installation from the `environment.yml`. It must be the newest version.